### PR TITLE
Reuse validation message component in upload UI, refine error message (SCP-4119)

### PR DESCRIPTION
**Pull request by eweitz**
This refactors `FileUploadControl.js` to use `ValidationMessage`, the newly resurrected component used in the sync UI.

This improves maintainability by replacing duplicate code with a more general approach.  It also fixes something of a bug in an error message, where the shown message could be much too long.

To test:
* In upload UI, select errant file
* Confirm that error message includes a note on how to email support, including the study's accession

This satisfies SCP-4119.

### CODE REVIEW

**Review by bistline — APPROVED** [review]

File: app/javascript/lib/validation/shared-validation.js [hunk 1]
```diff
@@ -93,8 +93,9 @@ export function validateUniqueCellNamesWithinFile(line, isLastLine, dataObj) {
   if (isLastLine && dataObj.duplicateCellNames.size > 0) {
     const nameTxt = (dataObj.duplicateCellNames.size > 1) ? 'duplicates' : 'duplicate'
     const dupString = [...dataObj.duplicateCellNames].slice(0, 10).join(', ')
-    const msg = `Cell names must be unique within a file. 
-      ${dataObj.duplicateCellNames.size} ${nameTxt} found, including: ${dupString}`
+    const msg =
```
Comment (devonbush) [inline app/javascript/lib/validation/shared-validation.js:96]: Why are we getting rid of multiline strings?

**Review by devonbush — APPROVED** [review]
Looks good, just one question about multiline template strings

File: app/javascript/lib/validation/shared-validation.js (hunk 1 above)
Comment (eweitz) [reply app/javascript/lib/validation/shared-validation.js:96]: It made the related test fail.  Multiline template string get evaluated as having multiple lines, yet remain shown as one line in the web UI.

```
  ● Client-side file validation › catches duplicate cell names in cluster file

    expect(received).toEqual(expected) // deep equality

    - Expected  - 1
    + Received  + 2

    - Cell names must be unique within a file. 1 duplicate found, including: CELL_0001
    + Cell names must be unique within a file. 
    +       1 duplicate found, including: CELL_0001

      129 |     expect(errors).toHaveLength(1)
      130 |     expect(errors[0][1]).toEqual('content:duplicate:cells-within-file')
    > 131 |     expect(errors[0][2]).toEqual('Cell names must be unique within a file. 1 duplicate found, including: CELL_0001')
          |                          ^
      132 |   })
      133 |
      134 |   it('catches duplicate cell names in expression matrix file', async () => {

      at Object.<anonymous> (test/js/lib/validate-file-content.test.js:131:26)

```

File: app/javascript/components/validation/ValidationMessage.js [hunk 2]
```diff
@@ -39,13 +39,13 @@ function Summary({ msgs, issueCategory, fileName=null, isSync=false }) {
  * TODO (SCP-4119): Integrate this component into FileUploadControl.js
```
Comment (ehanna4) [inline app/javascript/components/validation/ValidationMessage.js:39]: Can remove this todo now

Comment (eweitz) [reply app/javascript/components/validation/ValidationMessage.js]: Good catch!  Done in 8851ace.

**Pull request merged**